src/js/components/stock-movement-wizard/request/EditPage.jsx

```diff
@@ -97,12 +97,19 @@ const AD_HOCK_FIELDS = {
         defaultMessage: 'Product name',
         attributes: {
           formatValue: value => (
-            <span className="d-flex align-items-center">
-              <span className="text-truncate">
-                {value.name || ''}
+            <Tooltip
+              html={<div className="text-truncate">{value.name}</div>}
+              theme="dark"
+              disabled={!value.translatedName}
+              position="top-start"
+            >
+              <span className="d-flex align-items-center">
+                <span className="text-truncate">
+                  {value.translatedName ?? value.name}
+                </span>
+                {renderHandlingIcons(value ? value.handlingIcons : null)}
               </span>
-              {renderHandlingIcons(value ? value.handlingIcons : null)}
-            </span>
+            </Tooltip>
```

Comment:
generally I don't mind it, since it is not a big deal here, but be aware that making the `Tooltip` disabled instead of just making:
```
if (value.translatedName) { 
 return <Tooltip>...</Tooltip>
}
return <span>...</span>
```
you still render to DOM some additional `Tooltips` that are never actually used